keploy.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+
+installKeploy (){
+    IS_CI=false
+    for arg in "$@"
+    do
+        case $arg in
+            -isCI)
+                IS_CI=true
+                shift
+            ;;
+            *)
+            ;;
+        esac
+    done
+
+    install_keploy_arm() {
+        curl --silent --location "https://github.com/keploy/keploy/releases/latest/download/keploy_linux_arm64.tar.gz" | tar xz -C /tmp
+
+        sudo mkdir -p /usr/local/bin && sudo mv /tmp/keploy /usr/local/bin
+        alias keploy='sudo -E env PATH=$PATH keploy'
```

Comment:
why is this alias not set in bashrc or zshrc ? This alias will vanish if user opens new termianl right..?

Reply:
Have set it in bashrc and zshrc depending on the bash file